README.md

```diff
@@ -1,73 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Math Magicians
 
-# Project Name
+Its first reate projject.
 
-> Description the project.
+## Available Scripts
 
+In the project directory, you can run:
 
-## Built With
+### `npm start`
 
-- Major languages
-- Frameworks
-- Technologies used
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-## Live Demo (if available)
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
 
```

Comment:
- [ ] Please add relevant details to the readme file a how to clone and install packages and start the app.

Reply:
@dasileker 
Thanks for reviewing my code. I will do it.  😊